### PR TITLE
Reenable s3cmd

### PR DESCRIPTION
**Pull request by m-kuhn**

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 19ae1ba5c7a3c071c8df538f652067b5d9c7fc45:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1870-8ee3e49df75206da5bcc0958a72e2892c7249758-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1870-8ee3e49df75206da5bcc0958a72e2892c7249758-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1870-8ee3e49df75206da5bcc0958a72e2892c7249758-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1870-8ee3e49df75206da5bcc0958a72e2892c7249758-x86.apk)

File: scripts/ci/upload_artifacts.sh [hunk 1]
```diff
@@ -17,9 +17,9 @@ then
   sudo mv build-${ARCH}/android-build/build/outputs/apk/release/android-build-release-signed.apk /tmp/${FILENAME}
   echo "${S3CFG}" > ~/.s3cfg
   s3cmd put --acl-public /tmp/${FILENAME} s3://qfieldapks/ci-builds/${FILENAME}
-  echo -e "\e[31Uploaded to https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/${FILENAME} \e[0m"
-#  s3cmd modify --add-header=content-type:application/vnd.android.package-archive s3://qfieldapks/ci-builds/${FILENAME}
-#  echo -e "\e[31ACL modified \e[0m"
+  echo -e "\e[31mUploaded to https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/${FILENAME} \e[0m"
+  s3cmd modify -v -d --add-header=content-type:application/vnd.android.package-archive s3://qfieldapks/ci-builds/${FILENAME}
```
Comment (mbernasocchi) [inline scripts/ci/upload_artifacts.sh:21]: maybe remove -v before merging?

**Review by mbernasocchi — APPROVED** [review]

File: scripts/ci/upload_artifacts.sh (hunk 1 above)
Comment (m-kuhn) [reply scripts/ci/upload_artifacts.sh]: done

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 16e60beb3d041ed37fa5f806aacc4ef47a909136:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1870-a583536dbfb44ee286316fd28319e0768a1a6109-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1870-a583536dbfb44ee286316fd28319e0768a1a6109-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1870-a583536dbfb44ee286316fd28319e0768a1a6109-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1870-a583536dbfb44ee286316fd28319e0768a1a6109-x86.apk)

**Pull request merged**